tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -336,6 +335,13 @@ error: Expected type to be \\"num\\", but got \\"str\\" instead
    |              ^^^^ Expected type to be \\"num\\", but got \\"str\\" instead
 
 
+error: Variable cannot be reassigned from inflight
+   --> ../../../examples/tests/invalid/class.w:61:5
+   |
+61 |     this.y = 1;
+   |     ^^^^^^^^^^^ Variable cannot be reassigned from inflight
```

Comment:
This line already has a more specific error above:

```
"y" cannot be initialized in the inflight initializer
```

Technically y is not even reassignable at all, so `cannot be reassigned from inflight` is true but misleading.

Reply:
Good point. Looking into this.

Reply:
Fixed.